### PR TITLE
Widget Width Media Queries

### PR DESCRIPTION
**Pull request by wylie**
Adding a new file that has media queries for the _width.less file

### CODE REVIEW

File: less/_heading.less [hunk 1]
```diff
@@ -3,7 +3,7 @@
 .headings(6); // will error if more than 6
 
 .heading {
-	color: @heading-color;
+	color: @colorP2;
```
Comment (wylie) [inline less/_heading.less:6]: supposed to be colorP2

File: less/_width.mq.less [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		 
+		@list: 20, 25, 33, 40, 50, 60, 66, 75, 80;
+		@listLength: length(@list);
+
+		.loop(@arrayLength, @index: 1) when (@index =< @arrayLength) {
+			@arrayVal: extract(@list, @index);
+			@pcnt: percentage(@arrayVal / 100);
+
+			> .w@{arrayVal} {
+				display: block;
```
Comment (wylie) [inline less/_width.mq.less:13]: These act just like the layout columns

**Comment by amyh** [comment]
Looks good.

File: less/_width.mq.less (hunk 2 above)
Comment (gjjones) [inline less/_width.mq.less:13]: How does this differ from the layout columns then? Does it justify new styles?

Also, I <3 cats

**Comment by caseytrombley** [comment]
Looks great, approved ;)

File: less/_width.mq.less (hunk 2 above)
Comment (wylie) [inline less/_width.mq.less:13]: we use different classes for the _layout and the _width files

Comment (gjjones) [inline less/_width.mq.less:13]: I don't have any real arguments at the moment to keep this out, but I do question if we can't condense our use down to a single width generator between _layout and _width.

Comment (wylie) [inline less/_width.mq.less:13]: we don't have a loop for the page layout classes, but I am sure we could do some condensing with our page and widget layout styles.

I'm merging this

**Pull request merged**